python/idsse_common/idsse/common/http_utils.py

```diff
@@ -0,0 +1,217 @@
+"""Helper function for listing directories and retrieving s3 objects"""
+# -------------------------------------------------------------------------------
+# Created on Tue Feb 14 2023
+#
+# Copyright (c) 2023 Colorado State University. All rights reserved.             (1)
+# Copyright (c) 2023 Regents of the University of Colorado. All rights reserved. (2)
+#
+# Contributors:
+#     Geary J Layne (2)
```

Comment:
Might as well update this copyright date, and you don't need to give Geary partial credit for your class.